app/views/shared/processed_and_deleted/_index.html.slim

```diff
@@ -8,7 +8,7 @@
       table *{class: "#{state}-applications"}
         thead
           tr
-            th =t('processed_applications.table_header.applicant')
+            = table_header
```

Comment:
Can this be called explicitly? i.e. ReferenceHelper.table_header, it's not very clear where it points otherwise.